components/dashboard/src/components/workspaces.tsx

```diff
@@ -273,7 +273,7 @@ class Workspaces extends React.Component<WorkspacesProps, WorkspacesState> {
                 <Grid item xs={12}>
                     <Paper className="gc-message">
                         <Typography component="p" >
-                            Unused workspaces are automatically deleted after {this.state.configuration.daysBeforeGarbageCollection} days of inactivity. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more.</a>
+                            Unused workspaces are automatically deleted after {this.state.configuration.daysBeforeGarbageCollection} days of inactivity. To prevent a workspace from being deleted, you can pin it. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more.</a>
```

Comment:
**suggestion**: This will significantly increase the sentence length beyond the common standards for readability (50-70 characters per line). What do you think of updating this with something like the following?

We can also remove trailing comma from the _Learn more_ help link as in https://github.com/gitpod-io/gitpod/pull/2536.

```suggestion
                            Unpinned workspaces that have been stopped for for than {this.state.configuration.daysBeforeGarbageCollection} days will be automatically deleted. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more</a>
```

| BEFORE | AFTER | PROPOSED |  
|-|-|-|
| <img width="893" alt="alert-before" src="https://user-images.githubusercontent.com/120486/108119192-1d1a1f80-70a8-11eb-8eea-29d2b7f323a7.png"> | <img width="893" alt="alert-after" src="https://user-images.githubusercontent.com/120486/108119187-1be8f280-70a8-11eb-88ac-a0ccd0ab5358.png"> | <img width="893" alt="alert-proposed" src="https://user-images.githubusercontent.com/120486/108119196-1e4b4c80-70a8-11eb-9b1b-ad9eb567b445.png"> |